build/cli.cjs

```diff
@@ -1538,6 +1538,212 @@ async function importResponse(oldPtauFilename, contributionFilename, newPTauFile
 
 }
 
+/*
+    Copyright 2018 0KIMS association.
+    This file is part of snarkJS.
+    snarkJS is a free software: you can redistribute it and/or modify it
+    under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    snarkJS is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+    or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+    License for more details.
+    You should have received a copy of the GNU General Public License
+    along with snarkJS. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+// This is to import multiple contributions as a whole from the community ppot 
+// on top of the initial ptau file. Mostly the same as powersoftau_import.js
+// except that we don't verify if contributionPreviousHash in community ppot 
+// challenge file matches lastChallengeHash in the ptau file. 
+// Note that the powersoftau_verify won't succeed because multiple contributions 
+// are imported as a whole but the public key is not available. To verify the 
+// new ptau file, however, we can simply do a bellman export and compare it 
+// with the original challenge file.
+async function importResponseNoOrigin(curve, power, contributionFilename, newPTauFilename, name, importPoints, logger) {
```

Comment:
Document the parameters

Reply:
Doc added in src/powersoftau_import_no_origin.js. This file is auto generated.